### PR TITLE
Add Oval Charm

### PR DESCRIPTION
**Pull request by ghoulslash**
Adds the oval charm

## Description
Adds the oval charm to item_expansion

## **Discord contact info**
ghoulslash#3839

### CODE REVIEW

File: src/data/text/item_descriptions.h [hunk 1]
```diff
@@ -2793,3 +2793,9 @@ static const u8 sShinyCharmDesc[] = _(
 	"A charm that will\n"
     "raise the chance\n"
     "of Shiny Pokémon.");
+    
+static const u8 sOvalCharmDesc[] = _(
+    "Increases the\n"
+    "chance of finding\n"
+    "eggs at the daycare.");
```
Comment (AsparagusEduardo) [inline src/data/text/item_descriptions.h:2800]: pokeemerald-0
The text goes over the description area.

**Review by AsparagusEduardo — CHANGES_REQUESTED** [review]

File: src/data/text/item_descriptions.h (hunk 1 above)
Comment (ExpoSeed) [reply src/data/text/item_descriptions.h:2800]: suggestion: "Raises the chance" on the first line maybe?

Comment (ShinyDragonHunter) [reply src/data/text/item_descriptions.h:2800]: Increases the odds

Comment (AsparagusEduardo) [reply src/data/text/item_descriptions.h:2800]: image
That should work

**Review by ExpoSeed — APPROVED** [review]

**Review by AsparagusEduardo — APPROVED** [review]

**Pull request merged**